### PR TITLE
Blog app - create a data model

### PR DESCRIPTION
**Pull request by oluyaratosin123**
Here in this project I have done the following:
- Build your project schema.
   - Create and run the necessary migration files.
   - Table and column names should match the ERD diagram.
     - note: photo for users table should be a link to an image
   - Foreign keys should be included.
   - All columns that are foreign keys should have a corresponding index
   
   - railsMigrate

### CODE REVIEW

File: db/migrate/20220407130736_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, null: false, foreign_key:{ to_table: :users }
+      t.references :posts, null: false, foreign_key: { to_table: :posts }
```
Comment (MrBrN197) [inline db/migrate/20220407130736_create_comments.rb:6]: - [ ] Please note that just like the `:author` reference this line should reference a `:post` not a `:posts`. 
- [ ] [OPTIONAL] in this case you don't need to specify the `to_table` parameter. rails can figure out this relationship based on the reference name.

```suggestion
      t.references :post, null: false, foreign_key: true
```

File: db/migrate/20220407130737_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: { to_table: :posts }
```
Comment (MrBrN197) [inline db/migrate/20220407130737_create_likes.rb:5]: - [ ] [OPTIONAL] in this case you don't need to specify the `to_table:` parameter. rails can figure out this relationship based on the reference name.

```suggestion
      t.references :post, null: false, foreign_key: true
```

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @oluyaratosin123  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ All linter checks are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @oluyaratosin123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**